docs/nuget/local.md

```diff
@@ -0,0 +1,52 @@
+# Building Packages Locally
```

Comment:
If `Directory.Build.targets` is removed, can remove this. 

Was just thinking if we even need to mention how to build in the readme... it's all standard `dotnet` CLI after all.